### PR TITLE
allow case that wire separation is zero in one plane

### PR DESCRIPTION
**Pull request by mfittere**
I think you can just merge as the change is so small.

### CODE REVIEW

**Comment by kyrsjo** [comment]
Looks fine. Please bump the last digit of the version number and the date, and I'll merge.

**Comment by kyrsjo** [comment]
By the way, given how timezones and sleep works, you may end up being merged after the new Astuce: https://github.com/SixTrack/SixTrack/pull/239#discussion_r109468804
If so, please merge from the newest upstream master before bumping the version number.

Also, I ran the relevant tests for you, they were happy (as expected):
```
$ ctest -R wire -j 8
Test project /home/kyrsjo/code/SixTrack/SixTrack/SixTrack_cmakesix_BUILD_TESTING_defaultcompiler_defaultbuildtype
    Start 40: wire_thck4d
    Start 41: wire_thck6d
    Start 42: wire_thin4d
    Start 43: wire_thin6d
    Start 44: wire_thin6d_ibeco0
1/5 Test #40: wire_thck4d ......................   Passed    5.88 sec
2/5 Test #42: wire_thin4d ......................   Passed    6.76 sec
3/5 Test #41: wire_thck6d ......................   Passed    7.32 sec
4/5 Test #43: wire_thin6d ......................   Passed   15.26 sec
5/5 Test #44: wire_thin6d_ibeco0 ...............   Passed   15.68 sec

100% tests passed, 0 tests failed out of 5

Label Time Summary:
fast    =  50.90 sec (5 tests)

Total Test time (real) =  15.97 sec
```

I also checked the manual, all good there.

File: SixTrack/sixtrack.s [hunk 1]
```diff
@@ -16782,8 +16782,9 @@ cc2008
      &'vert. tilt         = ',wire_tilty(j),' degrees'
 ! ignore wire if current, length or displacment are 0 or
 ! wire_flagco not set (case wire_flagco = 0)
+! for displacement only ignore if wire_dispx = wire_dispy = 0
             if( abs(wire_flagco(j)*wire_current(j)*wire_lint(j)
-     &*wire_lphys(j)*wire_dispx(j)*wire_dispy(j)).le.pieni ) then
+     &*wire_lphys(j)*(wire_dispx(j)+wire_dispy(j))).le.pieni ) then
```
Comment (kyrsjo) [inline SixTrack/sixtrack.s]: Could you please paranthesize this a*b*c*d... We don't want some binaries switching it into a marker and some not.

Comment (mfittere) [inline SixTrack/sixtrack.s]: Fixed! Please go ahead and merge. I also checked out the latest master.

**Comment by kyrsjo** [comment]
You can go first; please bump the version number and date in sixtrack.s; if you have the access right you can then merge it yourself.

**Pull request merged**